release-notes-9.2.1.md

```diff
@@ -0,0 +1,11 @@
+#### Version Number
+${version-number}
+
+#### New Features
+- None
+
+#### Patch Fixes Included
+- This release includes OS package updates only
```

Comment:
This is not true.  Don't write that there have only been OS updates when there have clearly been lots of Java dependency updates as well.